bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -8,11 +8,11 @@ metadata:
           "apiVersion": "rhpam.kiegroup.org/v1",
           "kind": "KogitoBuild",
           "metadata": {
-            "name": "ruleunit-quarkus-example"
+            "name": "process-quarkus-example"
```

Comment:
are you sure about that change ?

Reply:
yes, process are not yet supported, we did this same change on 1.13.
However, I didn't update it manually, just run the command to generated the bundle.